packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -53,6 +53,8 @@ export interface Props extends Themeable2 {
   className?: string;
   /** Color mode for coloring the value or the background */
   colorMode: BigValueColorMode;
+  /** Whether or not a horizontal gradient is applied to the spark line graph */
```

Comment:
This comment is a bit off, should be "applies to background" or something similar as the sparkline can still be disabled.

Reply:
Agreed. Updated.